PowerShellTools.HostService/ServiceManagement/Debugging/PowerShellDebuggingService.cs

```diff
@@ -762,6 +892,20 @@ public IEnumerable<CallStack> GetCallStack()
                             (string)psFrame.FunctionName.ToString(),
                             (int)psFrame.ScriptLineNumber));
                 }
+                else
+                {
+                    String currentCall = psobj.ToString();
+                    Match match = validStackLine.Match(currentCall);

```

Comment:
Correct. With the connection the runspace has in this scenario, the cmdlet to get the call stack returns us raw strings. And I should probably use lowercase string here.